scripts/update-apk.sh

```diff
@@ -26,17 +26,21 @@ fi
 find ../app/build/outputs -type f -name '*.apk' -exec cp -v {} . \;
 find ../app/build/outputs -type f -name '*.aab' -exec cp -v {} . \;
 
-if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-    for file in app*; do
-          cp $file eventyay-organizer-master-${file:4}
-    done
-fi
+for file in app*; do
 
-if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
-    for file in app*; do
-          mv $file eventyay-organizer-dev-${file:4}
-    done
-fi
+    if [ ${file} =~ ".aab" ]; then
```

Comment:
We need to add master/dev before it. aap == apk

Reply:
Updated.